src/main/java/taxi/controller/driver/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,25 @@
+package taxi.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.service.CarService;
+
+@WebServlet(urlPatterns = "/cars/my")
```

Comment:
I'd recommend you to use path sth like: `"/drivers/cars/my"`, cause this is related to driver